README.md

```diff
@@ -3,30 +3,27 @@
- 4. 🔄 Be ready to update and run the tool again in the future to pick up changes in output.
- 5. 📚 _If_ you want to publish the output of the tool, use a pre-release version and tag for now, until we can make stronger guarantees about the API of the output it generates.
-
-
-To track our status on Polymer elements, check out or [elements status tracker](./docs/polymer-3-element-status.md).
-For a list of all breaking changes, see our [Breaking Changes Changelog](./docs/breaking_changes.md).
-
-Good luck, the future is exciting! 😎 Please join us for discussion in the [`#modulizer` Slack channel](https://polymer.slack.com/messages/G6R11FXEC/).
+Convert Polymer 2.x projects to Polymer 3.x.
+.
+## Overview
 
+polymer-modulizer, aka Modulizer, converts Bower packages using HTML Imports to npm packages using JavaScript modules. It automatically rewrites code to upgrade your projects and elements. In fact, the Polymer team used modulizer to upgrade the Polymer Elements, and Polymer itself!
 
-## Overview
+Modulizer performs many different upgrade tasks, like:
+ * Detects which `.html` files are used as HTML Imports and move them to `.js`
+ * Rewrites `<link rel="import>` in HTML to `import` in JS.
+ * Removes "module wrappers" - IIFEs that scopes your code.
+ * Converts `bower.json` to `package.json`, using the corresponding packages on npm.
+ * Converts "namespace references" to the proper JS module import, ie: `Polymer.Async.timeOut` to `timeOut` as imported from `@polymer/polymer/lib/util/async`.
+ * Creates exports for values assigned to namespace referencs. ie, `Foo.bar = {...}` becomes `export const bar = {...}`
+ * Rewrites namespace objects - an object with many members intended to be used as a module-like object, to JS modules.
+ * Moves Polymer element templates from HTML, into a JS template string.
+ * Removes `<dom-module>`s if they only contained a template.
+ * Moves other generic HTML in the document into a JS string and creates it when the module runs.
 
-polymer-modulizer is designed to convert HTML documents, and especially those containing Polymer and Polymer elements, to JavaScript modules as automatically as possible.
+Modulizer then writes out a file that records what changes it made, so that it can be published to npm along with the new package, so that dependencies of your package can also automatically be upgraded with modulizer.
 
-polymer-modulizer tries to strike a balance between supporting the general semantics of HTML Imports and generating idiomatic JavaScript modules. It converts assignments to global namespace objects to module exports, and references to namespaces as imported names.
+Modulizer will also install dependencies from npm, run tests, push to a git branch and publish to npm for you! And modulizer has a "workspace" mode where it will do this for entire collections of packages at once.
 
```

Comment:
Might add a note after this, or even add a new section here, like:

### Manual changes may be required

Manual changes may be required for some projects. In particular, if your project uses `Polymer.importHref`,
you'll need to convert these references manually to use the dynamic `import()` statement.

(example)

Reply:
Would be helpful to me, BTW, to understand more about this, because I don't know why it's manual, and/or what you need to change. The way it's been changed in PSK, for example, is quite different and I don't know why.

Reply:
done, and the 3 item list is basically what I'm aware of